doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -11,92 +11,83 @@ These tactics use the equality :g:`eq:forall A:Type, A->A->Prop` defined in
 file ``Logic.v`` (see :ref:`coq-library-logic`). The notation for :g:`eq T t u` is
```

Comment:
This is inconsistent with the actual implicit arguments (`A` is implicit in the type of `eq`, not just in the notation). The paragraph could be slightly amended to be more consistent with the reality.

Reply:
Could you suggest wording?  I'm not sure what's needed.